miden-crypto/src/word.rs

```diff
@@ -86,15 +91,15 @@ impl Digest for RpoDigest {
     }
 }
 
-impl Deref for RpoDigest {
-    type Target = [Felt; DIGEST_SIZE];
+impl Deref for Word {
+    type Target = [Felt; WORD_SIZE];
 
     fn deref(&self) -> &Self::Target {
         &self.0
     }
 }
 
-impl Ord for RpoDigest {
+impl Ord for Word {
```

Comment:
I would remove the lines below

```
// the endianness is irrelevant here because since, this being a cryptographically secure
// hash computation, the digest shouldn't have any ordered property of its input.
```

Reply:
Removed